nilearn/tests/test_signal.py

```diff
@@ -627,21 +697,19 @@ def test_clean_confounds_errros(signals):
     with pytest.raises(TypeError, match="confound has an unhandled type"):
         clean(signals, confounds=[None])
 
-    with pytest.raises(
-        ValueError, match="Confound signal has an incorrect length"
-    ):
+    msg = "Confound signal has an incorrect length."
+    with pytest.raises(ValueError, match=msg):
         clean(signals, confounds=np.zeros(2))
-    with pytest.raises(
-        ValueError, match="Confound signal has an incorrect length"
-    ):
+    with pytest.raises(ValueError, match=msg):
         clean(signals, confounds=np.zeros((2, 2)))
-    with pytest.raises(
-        ValueError, match="Confound signal has an incorrect length."
-    ):
+    with pytest.raises(ValueError, match=msg):
```

Comment:
This part could also be parameterized.

Reply:
yeah I thought about it but it felt like it would affect the readability of the test.